db/migrate/20221203071617_create_comments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.integer :comments_counter
```

Comment:
- [ ] As per the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), the Comments table does not have the field `comments_counter`. Kindly remove this field to respect the project requirements.